src/Tribe/Commerce/PayPal/Gateway.php

```diff
@@ -126,23 +126,24 @@ public function add_to_cart() {
 		$notify_url = tribe_get_option( 'ticket-paypal-notify-url', home_url() );
 
 		/**
-		 * Filters the notify URL.
+		 * Filters the Notify URL.
 		 *
 		 * The `notify_url` argument is an IPN only argument specifying the URL PayPal should
 		 * use to POST the payment information.
 		 *
-		 * @since 4.7
+		 * @link  https://developer.paypal.com/docs/classic/paypal-payments-standard/integration-guide/Appx_websitestandard_htmlvariables/
 		 *
-		 * @see  Tribe__Tickets__Commerce__PayPal__Handler__IPN::check_response()
-		 * @link https://developer.paypal.com/docs/classic/paypal-payments-standard/integration-guide/Appx_websitestandard_htmlvariables/
+		 * @see   Tribe__Tickets__Commerce__PayPal__Handler__IPN::check_response()
 		 *
-		 * @param string $notify_url
-		 * @param WP_Post $post The post tickets are associated with
-		 * @param array $product_ids An array of ticket post IDs that are being added to the cart
+		 * @since 4.7
+		 *
+		 * @param string  $notify_url  The Notify URL.
+		 * @param WP_Post $post        The post tickets are associated with.
+		 * @param array   $product_ids An array of ticket post IDs that are being added to the cart.
 		 */
 		$notify_url = apply_filters( 'tribe_tickets_commerce_paypal_notify_url', $notify_url, $post, $product_ids );
 
-		$custom_args = array( 'user_id' => get_current_user_id(), 'tribe_handler' => 'tpp', 'pid' => $post->ID );
+		$custom_args = [ 'user_id' => get_current_user_id(), 'tribe_handler' => 'tpp', 'pid' => $post->ID ];
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: e798af3b852166ff2c5f5ef158fd83fe</sub>